src/types/api.js

```diff
@@ -113,6 +113,8 @@
  * @prop {Organization} organization - nb. This field is nullable in the API, but
  *   we assign a default organization on the client.
  * @prop {GroupScopes|null} scopes
+ * @prop {Object} links
+ *   @prop {string} links.html
```

Comment:
The `html` property should be optional. It is not currently present for groups in third-party authorities for example.